op_crates/webidl/00_webidl.js

```diff
@@ -802,6 +802,50 @@
     throw new TypeError("Illegal constructor");
   }
 
+  function mixinPairIterable(name, prototype, dataSymbol, keyKey, valueKey) {
```

Comment:
This is going to replace `DomIterableMixin` in a follow-up PR. (`03_dom_iterable.js` will be removed).